packages/liquidator/test/index.js

```diff
@@ -464,7 +455,8 @@ contract("index.js", function(accounts) {
         // To create an error within the liquidator bot we can create a price feed that we know will throw an error.
         // Specifically, creating a uniswap feed with no `sync` events will generate an error. We can then check
         // the execution loop re-tries an appropriate number of times and that the associated logs are generated.
-        uniswap = await UniswapMock.new();
+        const uniswap = await UniswapMock.new();
+        await uniswap.setTokens(collateralToken.address, collateralToken.address);
```

Comment:
Quote token and base token are the same?

Reply:
I'm gonna remove the base and quote nomenclature but I'll make it more clear that for this specific test it's just arbitrary and I just need to initiate these values